cli/kaos_cli/facades/backend_facade.py

```diff
@@ -22,35 +23,102 @@ def is_cloud_provider(cloud):
 class BackendFacade:
     """
     This class should handle all backend related configuration and settings.
-
     """
 
     def __init__(self, state_service: StateService, terraform_service: TerraformService):
         self.state_service = state_service
         self.tf_service = terraform_service
 
+    @property
+    def active_context(self):
+        return self.state_service.get(ACTIVE, 'environment')
+
     @property
     def url(self):
-        return self.state_service.get(BACKEND, 'url')
+        return self.state_service.get_section(self.active_context, BACKEND, 'url')
 
     @property
     def user(self):
-        return self.state_service.get(BACKEND, 'user')
+        return self.state_service.get(DEFAULT, 'user')
 
     @property
     def token(self):
-        return self.state_service.get(BACKEND, 'token')
+        return self.state_service.get_section(self.active_context, BACKEND, 'token')
 
     @property
     def kubeconfig(self):
-        return self.state_service.get(INFRASTRUCTURE, 'kubeconfig')
+        return self.state_service.get_section(self.active_context, INFRASTRUCTURE, 'kubeconfig')
 
     def init(self, url, token):
         if not self.state_service.is_created():
             self.state_service.create()
         self.state_service.set(BACKEND, url=url, token=token)
+        self.state_service.set_section(REMOTE, BACKEND, url=url, token=token)
+
         self.state_service.write()
 
+    def list(self):
+        try:
+            contexts = self.state_service.get(CONTEXTS, 'environments')
+            # list_contexts = contexts.split(',')
```

Comment:
If this line is not needed, then please delete it.

Reply:
Yes, has been removed